addons/l10n_in_pos/static/src/overrides/models/order.js

```diff
@@ -6,6 +6,11 @@ import { patch } from "@web/core/utils/patch";
 patch(Order.prototype, {
     export_for_printing() {
         const result = super.export_for_printing(...arguments);
+        if (this.pos.company.country.code === 'IN') {
+            result.tax_details.forEach((tax)=>{
+                tax.tax.letter = tax.tax.description.substring(0, 4)
+            })
+        }
```

Comment:
Well I wanted to do it from `tax_group_id.name` but that would be take repetitive load of data where I have to load again account_tax_group_by_id but we can do it easily saas-17.1 this is `17.0` specific PR

Reply:
May I know where in the source code that makes sure that the `description` contains the characters we need here?

Reply:
https://github.com/odoo/odoo/blob/17.0/addons/l10n_in/data/template/account.tax-in.csv#L383